log/gps/gps_mgl_enigma_waypoint.ksy

```diff
@@ -0,0 +1,134 @@
+    type: waypoint
+    repeat: eos
+    doc: There is no demand that waypoints are stored in any particular order in the waypoint file. They might be sorted by short name or waypoint type but this is of little consequence. The order of waypoints in the file matters only for route files. Route waypoints are stored in the file in order of their appearance in the route. A route should have at least one waypoint.
+
+types:
+  waypoint:
+    seq:
+      - id: lat
+        type: sph_coord
+        doc: N is positive
+      - id: lon
+        type: sph_coord
+        doc: E is positive
+      - id: data_raw
+        type: u4
+      - id: type
+        type: u1
+        enum: type
+      - id: name_short
+        type: dc_str(6)
+        doc: This entry is used as “key” and in aviation terms would contain the short identifier of an airfield or navaid beacon. This key is used to lookup related data in other databases, for example an airport data base containing details of an airport such as frequencies and runway headings.
+      - id: name_long
+        type: dc_str(27)
+        doc: |
+          The long name contains a human readable description of the waypoint.
+          For example: “Los Angeles International”. The long name is arbitrary and not used as search key for any purpose."
+
+    instances:
+      is_data_frequency:
+        value: 25 >= type.to_i and type.to_i >= 9
+      altitude:
+        value: "((data_raw < 0x8FFFFFFF)?data_raw:-0x8FFFFFFF - data_raw)"
+        doc: in feet
+        if: not is_data_frequency
+      frequency:
+        value: data_raw
+        doc: in kHz
+        if: is_data_frequency
+
+  dc_str:
+    params:
+      - id: max_size
+        type: u1
+    seq:
+      - id: size
+        type: u1
+      - id: value
+        type: str
+        size: value_size
+      - id: garbage
+        type: str
+        size: max_size - value_size
+    instances:
+      value_size:
+        value: max_size > size ? size : max_size
+
+  sph_coord:
+    seq:
+      - id: raw
+        type: s4
+    instances:
+      value:
+        value: raw / 180000
+
+enums:
+  type:  # Contact info@MGLAvionics.co.za if you add any waypoint type, so they can update their PDF
+    0: unspecified
+    1:
+      id: aitport_minor
+      doc: Typical assignment for medium sized airports
+    2:
+      id: aitport_major
+      doc: Typical assignment for large and international airports
+    3:
+      id: seaplane_base
+    4:
+      id: airfield
+      doc: Typical assignment for smaller municipal airfields, glider fields etc
+    5:
+      id: airfield_private
+    6:
+      id: ultralight_field
+    7:
+      id: intersection
+      doc: reporting point, boundary crossing
+    8: heliport
+    9: tacan
+    10: ndb_or_dme
+    11: ndb
+    12: vor_or_dme
+    13: vortac
+    14: fan_marker
+    15: vor
+    16: rep_pt
+    17: lfr
+    18: uhf_ndb
+    19: m_ndb
+    20: m_ndb_or_dme
+    21: lom
+    22: lmm
+    23: loc_or_sdf
+    24: mls_or_ismls
```

Comment:
I assume these are common abbreviations in the context of aviation and don't need to be explained further? Maybe for clarity it would be good to at least say what each abbreviation stands for, so that it's easier to look them up.

Reply:
Added some `doc-ref`s.